Dockerfile

```diff
@@ -53,8 +55,6 @@ COPY --from=build --chown=appuser:appgroup \
 COPY --from=build --chown=appuser:appgroup \
         /app/node_modules ./node_modules
 
-RUN npm prune --production
```

Comment:
what's the reason for moving this ? doesn't this mean that the node_modules on the final image contains the dev dependencies?